psasim/test/Makefile

```diff
@@ -13,10 +13,11 @@ GENERATED_H_FILES =	./psa_manifest/manifest.h \
 all: $(TEST_BIN)
 
 psa_client: $(GENERATED_H_FILES)
+psa_client: client.c
 	$(CC) $(INCLUDE) $(CFLAGS) client.c $(LIB) -o $@
 
 psa_partition: $(GENERATED_H_FILES)
-psa_partition: psa_ff_bootstrap_TEST_PARTITION.c
+psa_partition: psa_ff_bootstrap_TEST_PARTITION.c server.c
```

Comment:
Actually, this isn't quite right - if `server.c` changes then we should re-run `psa_autogen.py` (since that script reads all the files in the directory, and gets the entry point from it).

`psa_autogen.py` then re-writes `psa_ff_bootstrap_TEST_PARTITION.c`, and we then want to compile it.

So the `server.c` dependency should really be for the rule that runs `psa_autogen.py` (i.e. at (new) line 23)